mitodl/mitxpro
# Registration form - Step 2

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #190

#### What's this PR do?
- Adds step 2 of the registration form
- Adds a `Profile` model to store the registration details

#### How should this be manually tested?
Go through the registration process. 
After completing Step 1, you should be directed to the Step 2 form.  
Make sure the form works as expected. Required fields are gender, birthyear, company,  job title.
After submitting Step 2, you should be directed back to the main page.  Check in Django admin that your user has a profile with the correct details.

#### Screenshots (if appropriate)
![Screen Shot 2019-05-09 at 11 03 30 AM](https://user-images.githubusercontent.com/187676/57465326-63dcb380-724c-11e9-95a4-70a16e7a44ae.png)

![Screen Shot 2019-05-09 at 11 04 33 AM](https://user-images.githubusercontent.com/187676/57465334-67703a80-724c-11e9-8083-c5f93477f385.png)
